Source/WebCore/platform/network/cocoa/NetworkStorageSessionCocoa.mm

```diff
@@ -137,21 +146,21 @@ - (CFURLRef)_cfurl;
         }
     }
 
+    auto aggregator = CallbackAggregator::create([this, weakThis = WeakPtr { *this }, completionHandler = WTFMove(completionHandler), newCookiesToAdd = WTFMove(newCookiesToAdd)] () mutable {
+        if (!weakThis)
```

Comment:
Not particularly harmful to have this, but do we need a weakThis? If we need one, should we turn it into a strong this?

Reply:
I'm going to capture a RetainPtr of nsCookieStorage() instead of weakThis.  That seems nicer.